src/edit/Toolbar.vue

```diff
@@ -247,16 +247,6 @@ $font-family-secondary: Roboto, Helvetica, Arial, sans-serif;
     width: 20px;
     height: 20px;
     font-size: 20px;
-
-    &::before {
```

Comment:
I'm not sure why this style was needed before, but it is causing that tooltip icon is not positioned as other icons so I just removed it 
@vladimyr @zcuric

Reply:
All good.